src/OVAL/public/oval_definitions.h

```diff
@@ -1828,15 +1901,15 @@ struct oval_component *oval_variable_get_component(struct oval_variable *);	//ty
  * It should be freed after use by the calling application.
  * @memberof oval_variable
  */
-struct oval_iterator *oval_variable_get_possible_values(struct oval_variable *variable);
+struct oval_variable_possible_value_iterator *oval_variable_get_possible_values(struct oval_variable *variable);
```

Comment:
Changing signatures of public functions breaks API and possibly ABI. We simply can't do that in maintenance branches, no exceptions. Distributions wouldn't be able to easily update to lates openscap upstream if we broke API/ABI. Even if the function itself was broken the best we can do is create a function with a different name that does it right and deprecate the broken function in maintenance branch, to be removed in the future.